pages/datasets.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: datasets
+title: Datasets
+nav-title: Datasets
+order: 0
```

Comment:
Make 1 and update the pubs page to be 2.
(We want to reserve 0 for possible dynamic things.)